src/validation.ts

```diff
@@ -655,6 +657,14 @@ export function validateWalletInit(
     name: 'walletInit.showTorusButton',
     value: showTorusButton,
     type: 'boolean',
+    optional: false
```

Comment:
Hey @chaitanyapotti did you mean to make the `showTorusButton` parameter no longer optional? This would be a breaking change, so will take some before we could get this merged in it's current state. I would recommend leaving this as an optional parameter if it doesn't break anything on your end.

Reply:
My bad.! It should still be optional. I was testing and made it non-optional.
I reverted it now.

Reply:
No worries! Will get this merged soon 👍